src/mscorlib/src/System/Text/ASCIIEncoding.cs

```diff
@@ -78,12 +80,157 @@ public override unsafe int GetByteCount(char* chars, int count)
             return EncodingForwarder.GetByteCount(this, chars, count);
         }
 
-        public override int GetBytes(String chars, int charIndex, int charCount,
-                                              byte[] bytes, int byteIndex)
+        public unsafe override byte[] GetBytes(String s)
+        {
```

Comment:
The encoding classes are not sealed. These changes have potential to break people who inherit from them. @tarekgh Have you evaluated the breaking impact of this change?